spec/support/factory_girl.rb

```diff
@@ -7,14 +7,14 @@
     password_confirmation 'password'
   end
 
+  factory :category do
+    name 'food'
+  end
+
   factory :item do
     sequence(:name) { |n| "Item #{n}" }
     image 'https://www.image.png'
     description 'item description'
-    purchase_url 'https://www.purchase.com'
```

Comment:
Below this line you can add category and then remove it from the 15 on your spec.